.travis.yml

```diff
@@ -30,69 +49,57 @@ matrix:
       env: TF_PACKAGE=tensorflow==1.1.0 KERAS_PACKAGE=keras==2.0.0 MPI=MPICH
     - python: "3.6"
       env: TF_PACKAGE=tensorflow==1.1.0 KERAS_PACKAGE=keras==2.0.0 MPI=MPICH
-    - python: "3.6"
+    - python: "3.4"
       env: TF_PACKAGE=tf-nightly KERAS_PACKAGE=git+https://github.com/fchollet/keras.git MPI=OpenMPI
 
 install:
   - |
-    if [ $MPI == "OpenMPI" ]; then
-      wget -O /tmp/openmpi-3.0.0-bin.tar.gz https://github.com/uber/horovod/files/1439728/openmpi-3.0.0-bin.tar.gz
-      pushd /usr/local && sudo tar -zxf /tmp/openmpi-3.0.0-bin.tar.gz && popd
-      export PATH=/usr/local/openmpi/bin:$PATH
+    if [[ $MPI == "OpenMPI" ]]; then
+      docker exec $CONTAINER /bin/sh -c "wget -O /tmp/openmpi-3.0.0-bin.tar.gz https://github.com/uber/horovod/files/1596799/openmpi-3.0.0-bin.tar.gz"
+      docker exec $CONTAINER /bin/sh -c "cd /usr/local && tar -zxf /tmp/openmpi-3.0.0-bin.tar.gz && ldconfig"
```

Comment:
does ldconfig take care of the PATH?

Reply:
I changed `openmpi-3.0.0-bin.tar.gz` to be installable directly into `/usr/local`. `mpirun` ends up being installed as `/usr/local/bin/mpirun`.